src/database/queries/task.js

```diff
@@ -0,0 +1,31 @@
+import knex from '../knex'
+import * as _ from './utilities'
+
+const addTask = attributes =>
```

Comment:
can you rename all these functions to not have `task` in them? They should just be called `add`,  `getAll`, `getBy`, `update`, `deleteAll` etc.